strapi/strapi
# [BUGFIX] - #17536

### What does it do?
Updated below server.js 

```js
 module.exports = ({ env }) => ({
  host: env('HOST', '0.0.0.0'),
  port: env.int('PORT', 1337),
  url: env('URL', 'http://localhost:1337/'), // Updated this line
  app: {
    keys: env.array('APP_KEYS'),
  },
  webhooks: {
    populateRelations: env.bool('WEBHOOKS_POPULATE_RELATIONS', false),
  },
});
```

### Why is it needed?

The issue being addressed is related to the incorrect construction of URLs during redirection when using AWS Cognito authentication. The original code added "https" twice in the URL, causing redirection problems.

### How to test it?

- Ensure you have the specified system information, Node.js version, NPM version, Strapi version, etc., set up.

- Use the provided URL to log in to AWS Cognito: http://localhost:1337/api/connect/cognito/

- Observe the redirection behavior and verify that the URL constructed is correct and doesn't have duplicated "https".

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
